packages/docregistry/src/default.ts

```diff
@@ -147,10 +147,17 @@ export class DocumentModel
     // A DocumentModel is a CodeCell and a nbmodel in one.
     const ynotebook = this.nbnotebook as nbmodel.YNotebook;
     if (!ynotebook.ymodel.has('source')) {
-      ynotebook.ymodel.set('source', new Y.Text(this.nbcell.getSource()));
+      ynotebook.ymodel.set('source', new Y.Text());
       ynotebook.ymodel.set('metadata', {});
       ynotebook.ymodel.set('cell_type', this.type);
     }
+
+    if (this.nbcell.getSource().length) {
```

Comment:
Good that you noticed this case. Let's remove this and fix this as we discussed in a separate PR

> An alternative method to fix this would be to add an additional parameter to the Websocket connection when this is opened as a text document

Reply:
Opened issue #49 to track this.

Reply:
Fixed in last commit. @hbcarlos Should another issue be opened for that?

Reply:
oh I have not seen your last comment. Resolving this conversation.